docs/self-managed/about-self-managed.md

```diff
@@ -13,7 +13,7 @@ However, Optimize documentation is available for both Camunda Platform 8 and Cam
 
 As an alternative to using Camunda Platform 8 through SaaS, you can host it yourself; we call this setup Camunda Platform 8 Self-Managed.
```

Comment:
This file needs a second look.

Reply:
The only changes in this PR are to Concepts content and links referencing Concepts content -- Self-Managed will take place next week 👍

Reply:
Ah ok, sorry for all the noise here. I now understand you were adjusting the URLs!